### PR TITLE
ftINIT

### PR DESCRIPTION
**Pull request by johan-gson**
### Main improvements in this PR:
* Added ftINIT to RAVEN, and updated the test cases accordingly.

**I hereby confirm that I have:**

- [X] Tested my code on my own machine
- [X] Followed the [development guidelines] (reasonably well)(https://github.com/SysBioChalmers/RAVEN/wiki/DevGuidelines).
- [X] Selected `devel` as a target branch
- [X] If needed, asked first in the [Gitter chat room](https://gitter.im/SysBioChalmers/RAVEN) about this PR

### CODE REVIEW

File: testing/unit_tests/tinitTests.m [hunk 1]
```diff
@@ -61,4 +61,507 @@ function testparsexpTask1List(testCase)
...
+    
+    prepDataTest5 = prepINITModel(testModel5, {}, {'R7';'R10'}, false, {}, 's');
+    tst1ResModel1 = ftINIT(prepDataTest5,arrayData1.tissues{1},[],[],arrayData1,{},getINITSteps(),true,true,testParams);
+    %We expect the 'true' path, i.e. through R2, not R9/R10 or R11-R14
+    verifyTrue(testCase, all(strcmp(tst1ResModel1.rxns,{'R1';'R2';'R4';'R6';'R7';'R8'})), 1)
+
+    %now add metabolite g
+    tst1ResModel1 = ftINIT(prepDataTest5,arrayData1.tissues{1},[],[],arrayData1,{'g'},getINITSteps(),true,true,testParams);
+    %We expect R2 to be replaced with R11 and R13
+    verifyTrue(testCase, all(strcmp(tst1ResModel1.rxns,{'R1';'R4';'R6';'R7';'R8';'R11';'R13'})), 1)
```
Comment (edkerk) [inline testing/unit_tests/tinitTests.m:428]: Here R2 is consistently not replaced. Running with Gurobi.

Comment (johan-gson) [reply testing/unit_tests/tinitTests.m:428]: Hmm, I will have to take a look at this. All passed when I did the pull request. Perhaps there is a mistake in this test case, or in the code.

Comment (edkerk) [reply testing/unit_tests/tinitTests.m:428]: It's the only (sub-)test that fails, the others pass fine.

Comment (johan-gson) [reply testing/unit_tests/tinitTests.m]: I have pushed some fixes for this now, the test case was simply not right.

Comment (edkerk) [reply testing/unit_tests/tinitTests.m]: Now it passes!

**Review by edkerk — APPROVED** [review]
Automated and manual tests all function. I cannot review whether the method is scientific sound, but this will also be reviewed in the future manuscript.

**Comment by JonathanRob** [comment]
Manual testing on `ftINIT` and older `tINIT` works fine for me (after some very minor fixes). Great work!

**Pull request merged**